.rustfmt.toml

```diff
@@ -1,4 +1,4 @@
-#
 # rustfmt nightly configuration options:
 # https://github.com/rust-lang/rustfmt/blob/master/Configurations.md
-#
+
+group_imports = "StdExternalCrate"
```

Comment:
Looks great!

What do you think about setting `imports_granularity = "Crate"` as well?

Reply:
I see 😅 alright. Let's talk about it later.